event-publisher/src/test/resources/uk/gov/ons/ctp/common/event/PackageFixture.NewCasePayloadContent.json

```diff
@@ -11,10 +9,6 @@
   "childMiddleName": "Rose",
   "childLastName": "Bloggs",
   "childDob": "2001-12-31",
-  "additionalInfo": "Class 2A",
```

Comment:
As discussed, this structure needs to match the structure with `sample`  and `sampleSensitive`  etc.  
We also need the consentGiven fields to output strings.  You could do this by changing the type, OR by using some Jackson annotation to convert (probably).
This hightlights how weak our EventPublisherTest tests are, since this was not picked up.

Reply:
Corrected